README.md

```diff
@@ -396,6 +396,16 @@ To build the containerized version, use either the Dockerfile `Dockerfile.armv7h
 docker buildx build --file Dockerfile.<ARCH> --tag acap-runtime:<ARCH>-containerized .
 ```
 
+## Building protofiles for Python
+
+The repository includes a Dockerfile (`Dockerfile.proto`) for building the APIs protofiles for Python. The Dockerfile generates the necessary Python files from the protobuf definitions, allowing gRPC communication with the ACAP Runtime service, without the need to build the protofiles in your application.
```

Comment:
Pointing out that copying is a/the way for applications.
```suggestion
The repository includes a Dockerfile (`Dockerfile.proto`) for building the APIs protofiles for Python. The Dockerfile generates the necessary Python files from the protobuf definitions, allowing gRPC communication with the ACAP Runtime service. This means that applications can copy these prebuilt files from ACAP Runtime container image instead of having to build the protofiles themselves.
```